### DISCUSSION TITLE
CodeSandBox preview not working.

### DISCUSSION THREAD

**Discussion by GJohnson22** [codesandbox/codesandbox-client#8439] [Q&A]
<img width="1289" alt="Screenshot 2024-05-01 at 8 50 57 AM" src="https://github.com/codesandbox/codesandbox-client/assets/159201101/097b839c-45cf-4be5-bd03-e85711a38d30">

I get a white preview panel every time. Any help will be appreciated.

**Comment by filipelima18** [comment]
Hi! The output of your code is shown in the console. To open it, click the Chrome DevTools icon on the top right of the preview, as shown below.
![image](https://github.com/codesandbox/codesandbox-client/assets/45261568/71043e1d-b960-4324-9584-5fffd6f03068)
